### PR TITLE
new command `into sqlite` allows you to take lists and create a sqlite database

### PR DESCRIPTION
**Pull request by fdncred**
# Description

This PR introduces a new command called `into sqlite`. It allows you to take lists of information and easily create a sqlite database with them. Here's a few examples.

## create database from a `ls`, creating the table called `main`
```sh
ls | into sqlite my_ls.db
```
image

## create database from a `ls` and naming the table
```sh
ls | into sqlite my_ls.db -t my_table_name
```
image

## create database from string literals tables
```sh
[one 2 5.2 six true 100mib 25sec] | into sqlite variety.db
```
image

# Tests

Make sure you've done the following:

- [ ] Add tests that cover your changes, either in the command examples, the crate/tests folder, or in the /tests folder.
- [ ] Try to think about corner cases and various ways how your changes could break. Cover them with tests.
- [ ] If adding tests is not possible, please document in the PR body a minimal example with steps on how to reproduce so one can verify your change works.

Make sure you've run and fixed any issues with these commands:

- [x] `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
- [x] `cargo clippy --workspace --features=extra -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
- [x] `cargo test --workspace --features=extra` to check that all the tests pass

### CODE REVIEW

**Comment by rgwood** [comment]
Very cool, thanks!

File: crates/nu-command/src/database/commands/into_sqlite.rs [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+                "Expected a list but instead received a {}",
+                input.get_type()
+            ),
+            span,
+        )),
+    }
+}
+
+// This is taken from to text local_into_string but tweaks it a bit so that certain formatting does not happen
+fn nu_value_to_string(value: Value, separator: &str, config: &Config) -> String {
```
Comment (elferherrera) [inline crates/nu-command/src/database/commands/into_sqlite.rs:234]: There is a into string function for value. No?

Comment (fdncred) [reply crates/nu-command/src/database/commands/into_sqlite.rs:234]: There are a bunch of these type of functions but I couldn't find one that did exactly what I needed to do.
